kategory-core/src/test/kotlin/kategory/data/MapKWTest.kt

```diff
@@ -56,6 +53,7 @@ class MapKWTest : UnitSpec() {
             }
         }
 
-        testLaws(TraverseLaws.laws(MapKW.traverse<String>(), MapKW.traverse<String>(), { a: Int -> mapOf<String, Int>("key" to a).k() }, EQ))
+        testLaws(TraverseLaws.laws(MapKW.traverse<String>(), MapKW.traverse<String>(), { a: Int -> mapOf<String, Int>("key" to a).k() }))
```

Comment:
WHERE IS YOUR EQ NOW? Love it.

Reply:
Should I do a cleanup run over all tests for this? I was planning to do it in the optics module in a separate PR since I also need to review the laws depending on `Gen`.